app/src/main/java/com/ryan/heroestopbuilds/MainActivity.java

```diff
@@ -212,21 +212,18 @@ public String onChildPress(String name) {
      */
     public String prettyPrinter(ArrayList<String> popularSkills) {
         //add our final list to a new list to be passed to MainActivity
-        String level1 =  "Level 1:     " + popularSkills.get(0);
-        String level4 =  "Level 4:     " + popularSkills.get(1);
-        String level7 =  "Level 7:     " + popularSkills.get(2);
-        String level10 = "Level 10:   " + popularSkills.get(3);
-        String level13 = "Level 13:   " + popularSkills.get(4);
-        String level16 = "Level 16:   " + popularSkills.get(5);
-        String level20 = "Level 20:  " + popularSkills.get(6);
-        ArrayList<String> finalList = new ArrayList<>();
-        finalList.add(level1);
-        finalList.add(level4);
-        finalList.add(level7);
-        finalList.add(level10);
-        finalList.add(level13);
-        finalList.add(level16);
-        finalList.add(level20);
+        String lgSpacing = String.format("%" + 3 + "s", "");
+        String smSpacing = String.format("%" + 1 + "s", "");
+        int count = 5;
```

Comment:
You could probably just hard code it honestly it would more accurately match the get(6) farther down